compiler/ballerina-lang/src/main/java/io/ballerina/projects/JarResolver.java

```diff
@@ -102,6 +102,12 @@ private void addPlatformLibraryPaths(PackageContext packageContext,
         }
     }
 
+    public Collection<Path> getJarFilePathsRequiredForTestRunner() {
```

Comment:
We shoudn't make changes to the JarResolver. Please move this to the RunTestsTask or TesterinaUtils

Reply:
Also, this only adds the platform libraries. We decided to add the thin jars of dependencies as well

Reply:
Removed the modification to JarResolver.